coalib/parsing/inline_comments.py

```diff
@@ -0,0 +1,30 @@
+import re
```

Comment:
and Makman2 [decided](https://gitter.im/coala/coala?at=5af2e45540f24c4304647b52) on `comments` for the module name